### DISCUSSION TITLE
How to access the administrative panel?

### DISCUSSION THREAD

**Discussion by ValeNaz** [globaleaks/globaleaks-whistleblowing-software#3618] [Q&A] [answered]
Hi, I'm trying to configure Globaleaks which I installed for the second time after formatting my machine, in my ubuntu server. I had already registered with my data on the same server and now when I open the .onion link to access the configuration this screen opens, as if I were a customer. How can I return to the configuration page?

Thanks in advance for the reply! 

![image](https://github.com/globaleaks/GlobaLeaks/assets/110800902/cb1ce0fd-76e0-490f-b462-125eb5307605)

**Comment by evilaliv3** [comment, marked as answer]
Hello @ValeNaz

GlobaLeaks is designed to simplify the user experience for whistleblower and this is why you do not find a login link on the homepage.

To access as administrator visit the url /login

**Reply by ValeNaz** [reply]
Thanks a lot!
